app/views/admin/consumer_apps/_form.html.erb

```diff
@@ -6,7 +6,7 @@
 
   <div class="crayons-field">
     <%= label_tag :platform, "Platform:", class: "crayons-field_label" %>
-    <%= select_tag :platform, options_for_select(ConsumerApp.platforms.invert, selected: @app.platform), class: "crayons-select", data: { "consumer-app-target" => "platform", "action" => "consumer-app#checkPlatform" } %>
+    <%= select_tag :platform, options_for_select(ConsumerApp.platforms.invert, selected: @app.platform, class: "crayons-select"), class: "crayons-select", data: { "consumer-app-target" => "platform", "action" => "consumer-app#checkPlatform" } %>
```

Comment:
**Aded crayons-select class**
In `app/views/admin/consumer_apps/_form.html.erb` for `Platform selection` the `crayons-select` class was getting used for internal items also but was removed by mistake in https://github.com/forem/forem/pull/18861/files 
Added `crayons-select` to internal items too.

So basically this was the exact code (i.e. using crayons-select for internal items as well as for overall item) before I started removing bootstrap but by mistake in #18861 it was removed and I was using this class at only one place and therefore I have added this class back for internal-items.

Although, UI wise there is not change.

Reply:
@rt4914 apologies I'm not sure I'm following.
When both classes are in the code like the change you've made above, one of them get ignored in the HTML because they are duplicated. This is what the HTML looks like for:

```
 <%= select_tag :platform, options_for_select(ConsumerApp.platforms.invert, selected: @app.platform, class: "crayons-select"), class: "crayons-select", data: { "consumer-app-target" => "platform", "action" => "consumer-app#checkPlatform" } %>

```

<img width="967" alt="Screenshot 2023-01-13 at 11 05 27" src="https://user-images.githubusercontent.com/2786819/212281068-946e9fa9-baca-4597-9b25-91700c807dc8.png">

Hence, I would think we need to remove the duplicate class that is not getting applied:

```suggestion
    <%= select_tag :platform, options_for_select(ConsumerApp.platforms.invert, selected: @app.platform), class: "crayons-select", data: { "consumer-app-target" => "platform", "action" => "consumer-app#checkPlatform" } %>
```

The form renders fine and still applies the class correctly.

<img width="1913" alt="Screenshot 2023-01-13 at 11 02 03" src="https://user-images.githubusercontent.com/2786819/212280503-094659c2-dd4f-4eca-a84f-1f0315b32dc9.png">

Perhaps I'm misunderstanding something though?

Reply:
Great. Thanks a lot for detailed explanation. It means that before working of removing bootstrap this implementation was incorrect at that time too.

I have removed the inner crayons-select class to fix duplicacy.

Reply:
It's a pleasure.

> It means that before working of removing bootstrap this implementation was incorrect at that time too.

Yip, exactly. Thank you for fixing it!